server.py

```diff
@@ -34,7 +34,7 @@ def changePath():
 @app.get("/question/<number>")
 def question(number):
     statement = questions[number].statement
-    return template("index.html", question_number=number, question=statement)
+    return template("templates/index.html", question_number=number, question=statement)
```

Comment:
From what I can [see here](https://bottlepy.org/docs/dev/_modules/bottle.html#template) I think this works without the templates prefix? Can you confirm that the templates prefix is needed? If not let's remove this change

Reply:
Done. I checked, it. The folder name was `views` in bottle and `templates` in Flask.
I earlier tested with `templates` name and it did not work so I made the commit.